make/mt-config/mt-presets.mk

```diff
@@ -99,19 +99,19 @@ else ifeq ($(PRESET), CC3200_TLS_SOCKET)
     XI_TARGET_PLATFORM = cc3200
 
 # -------------------------------------------------------
-# ST Micro STM32F4
-else ifeq ($(PRESET), STM32F4)
-    CONFIG = $(CONFIG_STM32F4)
+# ST Micro STM32FX
```

Comment:
good point. It can be good for Eval and Discovery. We don't have info about specifics now. This is the status of our knowledge and development process. But yes I agree, it is a bit odd, but we don't know more currently.